index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Kyam Harris | Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+	
+<body>
+	<header>
+		<nav class="container">
+			<ul class="navLinks">
+				<li><a href="#about">About</a></li>
+				<li><a href="portfolio.html" id="portFolio">Portfolio</a></li>
+				<li><a href="#foot">Contact</a></li>
+			</ul>
+			<div class="title">
+				<h1>Kyam Harris</h1>
+				<h4>Placeholder text, maybe skills and experience</h4>
+
+			</div>
+			<section class="social">
+				<a href="#" class="twitter"></a>
+				<a href="#" class="github"></a>
+				<a href="#" class="linkedIn"></a>
+			</section>
+			<div class="headImage">
```

Comment:
Maybe take out the text out the `<nav>` and put it in the `<header>`

Reply:
A bit of a headache with styling, going to leave as it works atm